charts/stable/synapse/questions.yaml

```diff
@@ -310,11 +306,6 @@ questions:
                 schema:
                   type: string
                   default: "Matrix <matrix@example.com>"
-              - variable: riotUrl
-                label: "Riot URL"
-                schema:
-                  type: string
-                  default: ""
```

Comment:
I can't see why is that removed?

Reply:
This wasn't used in any templates and from what I can tell it's supposed to be the deprecated `riot_base_url`. I can add it back if that's not the case.

Reply:
In that case it would be best to use the new config name and actually link it on the templates